scalahost/src/main/protobuf/semanticdb.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package org.scalameta.semantic.v1.proto;
```

Comment:
I opted for `org.scalameta` because scalapb generated the package name 

```
package `scala`.scalameta.semantic.v1...
```

It's a bug in scalapb, I guess I should report it, but I want the lazy route and used org.scalameta as a workaround.